flutter_app/lib/modules/get_airside_logs.dart

```diff
@@ -0,0 +1,13 @@
+import 'dart:io';
+
+class GetAirsideLogs{
+  final String pathToDirectory;
+
+  GetAirsideLogs({required this.pathToDirectory});
+
+  List <File> getFiles(){
```

Comment:
I'll leave it up to you, but if this class is going to be this simple you can consider making this function static and moving the `pathToDirectory` parameter to it. This means calling the function will be something like
`GetAirsideLogs.getFiles(PATH_TO_DIRECTORY);`

Reply:
i think i'll leave it as is, so that i can change it for when airside solidifies its logging system